### PR TITLE
[SFINT-4032] string review for sfint-3987

### PR DESCRIPTION
**Pull request by rtherien**
Issue: https://coveord.atlassian.net/browse/SFINT-4032

String review for [SFINT-3987](https://coveord.atlassian.net/browse/SFINT-3987)

### CODE REVIEW

**Review by aelfaleh — APPROVED** [review]
Thank you 🙇

File: packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 1]
```diff
@@ -236,7 +236,7 @@
         <value>Could not connect.</value>
         <language>en_US</language>
         <protected>false</protected>
-        <shortDescription>Could not connect.</shortDescription>
+        <shortDescription>Couldn't connect.</shortDescription>
```
Comment (lbergeron) [inline packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml:239]: Why did you change this description? `Could not connect.` seemed correct to me.

**Review by lbergeron — APPROVED** [review]

**Review by samisayegh — APPROVED** [review]

File: packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml (hunk 1 above)
Comment (rtherien) [reply packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml:239]: Nothing major, it's simply because we now accept contractions in our docs.
